airbyte-webapp/src/views/Connector/ServiceForm/ServiceForm.test.tsx

```diff
@@ -199,11 +202,13 @@ describe("Service Form", () => {
           formType="source"
           formValues={{ name: "test-name", serviceType: "test-service-type" }}
           onSubmit={(values) => (result = values)}
-          selectedConnectorDefinitionSpecification={{
-            connectionSpecification: schema,
-            sourceDefinitionId: "test-service-type",
-            documentationUrl: "",
-          }}
+          selectedConnectorDefinitionSpecification={
+            {
+              connectionSpecification: schema,
+              sourceDefinitionId: "test-service-type",
+              documentationUrl: "",
+            } as unknown as DestinationDefinitionSpecificationRead
```

Comment:
How come this is cast as `unknown` first?

Reply:
Casting to it directly causes a typescript error. I'll replace with
`// @ts-expect-error Partial objects for testing`